lib/jekyll/theme_builder.rb

```diff
@@ -8,7 +8,7 @@ class Jekyll::ThemeBuilder
   attr_reader :name, :path, :code_of_conduct
 
   def initialize(theme_name, opts)
-    @name = theme_name.to_s.tr(" ", "_").gsub(%r!_+!, "_")
+    @name = theme_name.to_s.tr(" ", "_").squeeze("_")
```

Comment:
I had no idea this method existed! Great find!

```text
>> "hello____friends___".squeeze("_")
=> "hello_friends_"
```

Reply:
Thanks.. we do use `squeeze` somewhere else in the codebase IIRC.. :smiley: